cli_tools/gce_vm_image_import/importer/disk_inspection_processor.go

```diff
@@ -0,0 +1,86 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package importer
+
+import (
+	"log"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/disk"
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+)
+
+type diskInspectionProcessor struct {
+	args          ImportArguments
+	diskInspector disk.Inspector
+}
+
+func (d *diskInspectionProcessor) process(pd persistentDisk) (persistentDisk, error) {
```

Comment:
minor: `d` sounds like it's short for disk or similar. Maybe `processor` or `dip`?

Reply:
changed to "p"